pandas/core/indexes/base.py

```diff
@@ -2441,10 +2441,22 @@ def _get_unique_index(self, dropna=False):
 
         Returns
         -------
-        loc : int if unique index, possibly slice or mask if not
-    """
+        loc : int if unique index, slice if monotonic index, else mask
 
-    @Appender(_index_shared_docs['get_loc'])
+        Examples
+        ---------
+        >>> unique_index = pd.%(klass)s(list('abc'))
```

Comment:
I don't think 'klass' substitution is needed for the example, just show it with base Index, logic of behaviour should be the same for all types (for other types of Index subclasses the 'abc' values might not be valid anyhow, and you get a 'wrong' example)

Reply:
Ok